<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1452</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabCleanRaw">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <attribute name="title">
         <string>Raw Data Cleanup</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QComboBox" name="comboBoxRawVar"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="widgetToolBar"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="widgetRawData"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0,0">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item alignment="Qt::AlignBottom">
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Outliers method:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButtonNone">
                 <property name="text">
                  <string>None</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">outliersgroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButtonIQR">
                 <property name="text">
                  <string>IQR</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">outliersgroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButtonZScore">
                 <property name="text">
                  <string>Z-score</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">outliersgroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="5">
                <widget class="QSpinBox" name="spinBoxStartMonth">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>12</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>M</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>D</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>End date:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Starting date:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QSpinBox" name="spinBoxStartDay">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>31</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="6">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="7">
                <widget class="QSpinBox" name="spinBoxStartYear">
                 <property name="minimum">
                  <number>1800</number>
                 </property>
                 <property name="maximum">
                  <number>2200</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QSpinBox" name="spinBoxEndDay">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>31</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="5">
                <widget class="QSpinBox" name="spinBoxEndMonth">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>12</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="7">
                <widget class="QSpinBox" name="spinBoxEndYear">
                 <property name="minimum">
                  <number>1800</number>
                 </property>
                 <property name="maximum">
                  <number>2200</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>D</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>M</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="6">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QRadioButton" name="radioButtonC">
                 <property name="text">
                  <string>Display in °C</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">temperatureGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButtonF">
                 <property name="text">
                  <string>Display in °F</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">temperatureGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButtonK">
                 <property name="text">
                  <string>Display in K</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">temperatureGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSelectPoints">
               <property name="text">
                <string>Select points</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonResetAll">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabImportCleaned">
        <attribute name="title">
         <string>Import cleanead measures</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Path to the cleaned measures</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditBrowseCleaned">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonBrowse">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="temperatureGroup"/>
  <buttongroup name="outliersgroup"/>
 </buttongroups>
</ui>
